### DISCUSSION TITLE
Disable VirusTotal and Release

### DISCUSSION THREAD

**Discussion by chapterjason** [objective-see/LuLu#569] [General]
Hello I have some questions before we want to use LuLu in prod.

- Is it possible to disable the VirusTotal Feature?
- Is there a release schedule or when can we expect that the next release might be happen? (As the removal of the crash reporting is important for us.)

Cheers!
Jason

**Comment by objective-see** [comment]
Sorry for the delayed response! 🫣

I will work on a feature to disable VT integration as I can see that value from a privacy point of view (though i should not that nothing is sent to VT unless you click on the VT button). 

In terms of release schedule, as LuLu is written by a single developer ...there is no established release schedule.

**Comment by objective-see** [comment]
Added, and will be in the next release :) 
https://github.com/objective-see/LuLu/commit/9bdfb4293a5d916723212fa7fd723223fd000ef3

**Comment by objective-see** [comment]
Closing this out, as this feature has been added, and now can be found the most recent releases:
![image](https://github.com/user-attachments/assets/d8047c6c-fee8-4364-a680-db4d8b9a15ac)
